holmes/config.py

```diff
@@ -185,13 +185,18 @@ def create_console_tool_executor(
                 default_toolsets, allowed_toolsets.split(",")
             )
 
-        # Enable all matching toolsets that have CORE or CLI tag
-        for toolset in matching_toolsets:
-            toolset.enabled = True
+        toolsets_loaded_from_config = self.load_custom_toolsets_config()
+        loaded_toolsets_from_env = None
+        if self.toolsets:
+            loaded_toolsets_from_env = self.load_toolsets_config(self.toolsets, "env")
+
+        # Enable all matching toolsets if no custom configs are provided!
+        if not toolsets_loaded_from_config and not loaded_toolsets_from_env:
```

Comment:
Please revert the enable of default/builtin tools according to the allowed_toolsets.